src/api/BrowserTypes.ts

```diff
@@ -0,0 +1,5863 @@
+     */
+    error?: FindInstancesErrors;
+    /**
+     * Used if the raiseIntent request was successfully resolved
+     */
+    intentResolution?: IntentResolution;
+    /**
+     * Used if a raiseIntent request requires additional resolution (e.g. by showing an intent
+     * resolver) before it can be handled.
+     */
+    appIntent?: AppIntent;
+}
+
+/**
+ * Identifies the type of the message and it is typically set to the FDC3 function name that
+ * the message relates to, e.g. 'findIntent', with 'Response' appended.
+ */
+
+/**
+ * A secondary response to a request to raise an intent used to deliver the intent result.
+ * This message should quote the original requestUuid of the raiseIntentRequest message in
+ * its `meta.requestUuid` field.
+ *
+ * A message from a Desktop Agent to an FDC3-enabled app responding to an API call. If the
+ * payload contains an `error` property, the request was unsuccessful.
+ */
+export interface RaiseIntentResultResponse {
+    /**
+     * Metadata for messages sent by a Desktop Agent to an App in response to an API call
+     */
+    meta: AddContextListenerResponseMeta;
+    /**
+     * A payload for a response to an API call that will contain any return values or an `error`
+     * property containing a standardized error message indicating that the request was
+     * unsuccessful.
+     */
+    payload: RaiseIntentResultResponsePayload;
+    /**
+     * Identifies the type of the message and it is typically set to the FDC3 function name that
+     * the message relates to, e.g. 'findIntent', with 'Response' appended.
+     */
+    type: "raiseIntentResultResponse";
+}
+
+/**
+ * A payload for a response to an API call that will contain any return values or an `error`
+ * property containing a standardized error message indicating that the request was
+ * unsuccessful.
+ */
+export interface RaiseIntentResultResponsePayload {
+    error?:        ResponsePayloadError;
+    intentResult?: IntentResult;
+}
+
+/**
+ * Identifies the type of the message and it is typically set to the FDC3 function name that
+ * the message relates to, e.g. 'findIntent', with 'Response' appended.
+ */
+
+/**
+ * Hello message sent by an application to a parent window or frame when attempting to
+ * establish connectivity to a Desktop Agent
+ *
+ * A message used during the connection flow for an application to a Desktop Agent in a
+ * browser window. Used for messages sent in either direction.
+ */
+export interface WebConnectionProtocol1Hello {
+    /**
+     * Metadata for this connection step message
+     */
+    meta: WebConnectionProtocol1HelloMeta;
+    /**
+     * The message payload, containing data pertaining to this connection step.
+     */
+    payload: WebConnectionProtocol1HelloPayload;
+    /**
+     * Identifies the type of the connection step message.
+     */
+    type: "WCP1Hello";
+}
+
+/**
+ * Metadata for this connection step message
+ *
+ * Metadata for a disconnection step message
+ */
```

Comment:
Is this metadata for both? I think 'this' is slightly confusing

Reply:
This is product of quicktype combining the schema comments for the two different schemas that metadata can take. Fixing that by moving the description from the two types onto the combination and talking about Web Connection Protocol messages instead